crypto/sha/sha512.c

```diff
@@ -432,6 +432,103 @@ static const SHA_LONG64 K512[80] = {
                                 : "=r"(ret)                     \
                                 : "r"(*((const SHA_LONG64 *)(&(x))))); ret; })
 #    endif
+#   elif (__riscv_zbkb || __riscv_zbb) && __riscv_xlen == 32
+#    define PULL64(x) ({ SHA_LONG64 ret;                                    \
+                        unsigned int *r=(unsigned int *)(&(ret));           \
```

Comment:
Formatting nit: spaces either size of the equals.
This repeats several times throughout.

Reply:
I just copied formatting from existing code (e.g. see line 411).
Should I add spaces around the '='?

Reply:
Yes please.

Reply:
Force pushed tabulation fixes